net/pfSense-pkg-Quagga_OSPF/files/usr/local/pkg/quagga_ospfd_raw.xml

```diff
@@ -80,6 +91,31 @@
 			<rows>30</rows>
 			<cols>65</cols>
 		</field>
+               <field>
+                        <fielddescr>ospf6d.conf</fielddescr>
+                        <fieldname>ospf6d</fieldname>
+                        <description>
+                                <![CDATA[
+                                Note: Once you click "Save" below, the assistant (in the "Global Settings" and "Interface Settings" tabs above) will be overridden 
+with w$
```

Comment:
Looks like a copy/paste word wrap issue here - should be on the previous line and some text appears to be missing.